### ISSUE TITLE
Do not require reference to System.ComponentModel.DataAnnotations

### ISSUE THREAD

**Issue by dswisher** [dswisher/yaclops#13]
To mark a positional parameter as required, the `[Required]` attribute from `System.ComponentModel.DataAnnotations` is used. In a new project, this is not included by default. To reduce dependencies, consider adding a `[Required]` attribute to `Yaclops.Attributes`.

**Comment by cj525** [comment]
+1

**Comment by dswisher** [comment]
In thinking about this a little more, I'm wondering if it would be better to use a new attribute, perhaps `[Mandatory]`, to avoid conflicts.

**Comment by cj525** [comment]
Agreed.  Another reason is that any external/reused attributes effectively hides the functionality from developer discovery via intellisense.
